js/private/js_run_devserver.bzl

```diff
@@ -4,103 +4,20 @@ load(":js_binary.bzl", "js_binary_lib")
 load(":js_binary_helpers.bzl", _gather_files_from_js_providers = "gather_files_from_js_providers")
 load("@bazel_skylib//lib:dicts.bzl", "dicts")
 
-_DOC = """Runs a devserver via binary target or command.
-
-A simple http-server, for example, can be setup as follows,
-
-```
-load("@aspect_rules_js//js:defs.bzl", "js_run_devserver")
-load("@npm//:http-server/package_json.bzl", http_server_bin = "bin")
-
-http_server_bin.http_server_binary(
-    name = "http_server",
-)
-
-js_run_devserver(
-    name = "serve",
-    args = ["."],
-    data = ["index.html"],
-    tool = ":http_server",
-)
-```
-
-A Next.js devserver can be setup as follows,
-
-```
-js_run_devserver(
-    name = "dev",
-    args = ["dev"],
-    command = "./node_modules/.bin/next",
-    data = [
-        "next.config.js",
-        "package.json",
-        ":node_modules/next",
-        ":node_modules/react",
-        ":node_modules/react-dom",
-        ":node_modules/typescript",
-        "//pages",
-        "//public",
-        "//styles",
-    ],
-)
-```
-
-where the `./node_modules/.bin/next` bin entry of Next.js is configured in
-`npm_translate_lock` as such,
-
-```
-npm_translate_lock(
-    name = "npm",
-    bins = {
-        # derived from "bin" attribute in node_modules/next/package.json
-        "next": {
-            "next": "./dist/bin/next",
-        },
-    },
-    pnpm_lock = "//:pnpm-lock.yaml",
-)
-```
-
-and run in watch mode using [ibazel](https://github.com/bazelbuild/bazel-watcher) with
-`ibazel run //:dev`.
-
-The devserver specified by either `tool` or `command` is run in a custom sandbox that is more
-compatible with devserver watch modes in Node.js tools such as Webpack and Next.js.
-
-The custom sandbox is populated with the default outputs of all targets in `data`
-as well as transitive sources & npm links.
-
-An an optimization, virtual store files are explicitly excluded from the sandbox since the npm
-links will point to the virtual store in the execroot and Node.js will follow those links as it
-does within the execroot. As a result, rules_js npm package link targets such as
-`//:node_modules/next` are handled efficiently. Since these targets are symlinks in the output
-tree, they are recreated as symlinks in the custom sandbox and do not incur a fully copy of the
-underlying npm packages.
-
-Supports running with [ibazel](https://github.com/bazelbuild/bazel-watcher).
-Only `data` files that change on incremental builds are synchronized when running with ibazel.
-"""
-
 _attrs = dicts.add(js_binary_lib.attrs, {
-    "tool": attr.label(
```

Comment:
Is this a breaking change?

Reply:
No. The macro the replaces the rule in the public API still has `tool`

Reply:
Yeah I see it now 😅